clinica/iotools/converters/ukb_to_bids/ukb_utils.py

```diff
@@ -94,6 +96,10 @@ def read_imaging_data(imaging_data_directory: PathLike) -> DataFrame:
     )
     dataframe = dataframe[filename.isin(file_mod_list)]
     filename = filename[filename.isin(file_mod_list)]
+    if dataframe.empty:
+        raise ValueError(
+            f"Dataset not found, or not handled by Clinica. Please check your data."
```

Comment:
I would be a bit more specific, maybe something like:

```python
f"No imaging data were found in the provided folder: {imaging_data_directory}. Please check your data."
```

WDYT?

Reply:
If we add the part "not handled by clinica" I agree. Since we do not handle all that is provided.

Reply:
> If we add the part "not handled by clinica" I agree. Since we do not handle all that is provided.

Sounds good to me !